controllers/cephcluster_controller.go

```diff
@@ -32,22 +37,56 @@ type CephClusterReconciler struct {
 	client.Client
 	Log    logr.Logger
 	Scheme *runtime.Scheme
+	cc     *hypersdsv1alpha1.CephCluster
 }
 
 // +kubebuilder:rbac:groups=hypersds.tmax.io,resources=cephclusters,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=hypersds.tmax.io,resources=cephclusters/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups="",resources=pods;configmaps,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=apps,resources=deployments;daemonsets;replicasets;statefulsets,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Do we need to watch deployments, daemonsets, replicasets and statefulsets too?

Reply:
Currently this is not needed.